pyproject.toml

```diff
@@ -1,25 +1,33 @@
 [tool.black]
 line-length = 79
 # note: for support "py39", we need to update black using poetry
-#       currently, it has conflicts with typing-extensions
+#       currently, it has conflicts with typing-extensions.
+#       Black has clonflicts with Jinja2 syntax in hooks/post_gen_project.py.
 target-version = ["py38"]
-force-exclude = """
-(
-  /(
-      docs/*
-    |.*\\.egg-info
-  )/
-)
-"""
+force-exclude = '''
+/(
+  |  docs/*
+  |  hooks/* 
+  |.*\\.egg-info
+)/
+'''
+
+[tool.flake8]
+max-line-length = 79
+# Ignoring these erros will prevent conflicts with Flake8 and Jinja2 syntax.
+per-file-ignores = [
```

Comment:
instead of excluding files per errors, please, exclude the folders.

maybe something like:
`exclude = .git,__pycache__,build,dist,docs,hooks,{{cookiecutter.project_slug}}`

PS: not sure if `{{cookiecutter.project_slug}}` will need some escape ... maybe `\{\{cookiecutter.project_slug\}\}`